site/content/docs/developing/content-delivery.ja.md

```diff
@@ -51,6 +51,23 @@ CloudFront で HTTPS を使用する場合、ロードバランサーの `http.c
 
 この 2 つの設定により、Copilot は CloudFront が [SSL/TLS 証明書](https://docs.aws.amazon.com/ja_jp/AmazonCloudFront/latest/DeveloperGuide/using-https-alternate-domain-names.html)を使用するように設定されます。これにより、ビューワーの証明書を検証し、HTTPS 接続を有効にすることができます。
 
-## Ingress 規制とは?
+## Ingress 制限とは?
 
-特定の送信元からの受信トラフィックを制限することができます。CloudFront の場合、Copilot は [AWS マネージドプレフィックスリスト](https://docs.aws.amazon.com/ja_jp/vpc/latest/userguide/working-with-aws-managed-prefix-lists.html)を使用して、CloudFront エッジロケーションに関連する CIDR IP アドレスのセットに許可されたトラフィックを制限します。`restrict_to.cdn: true` を指定すると、パブリックロードバランサーは完全にパブリックアクセスできなくなり、CloudFront 配信からのみアクセスできるようになり、Service に対するセキュリティ脅威から保護されます。
\ No newline at end of file
+特定の送信元からの受信トラフィックを制限できます。CloudFront の場合、Copilot は [AWS マネージドプレフィックスリスト](https://docs.aws.amazon.com/ja_jp/vpc/latest/userguide/working-with-aws-managed-prefix-lists.html)を使用して、CloudFront エッジロケーションに関連する CIDR IP アドレスのセットに許可されたトラフィックを制限します。`restrict_to.cdn: true` を指定すると、パブリックロードバランサーはパブリックアクセスできなくなり、CloudFront 配信からのみアクセスできるようになり、Service に対するセキュリティ脅威から保護されます。
```

Comment:
I prefer CloudFront ディストリビューション rather than CloudFront 配信 because [the documentation](https://aws.github.io/copilot-cli/ja/docs/developing/content-delivery/) uses it.

Reply:
OK. I will fixed it.